gapis/trace/android/adreno/profiling_data.go

```diff
@@ -32,14 +34,16 @@ var (
 		"ON s.track_id = t.id WHERE t.scope = 'gpu_render_stage'"
 	argsQueryFmt = "" +
 		"SELECT key, string_value FROM args WHERE args.arg_set_id = %d"
+	queueSubmitQuery = "" +
+		"SELECT submission_id FROM gpu_slice WHERE name = 'vkQueueSubmit' ORDER BY submission_id"
```

Comment:
Instead of looking for vkQueueSubmit slices in all the gpu slices, you need to limit this query to the vk events track.